docs/querying/basics.md

```diff
@@ -160,11 +168,15 @@ following units:
 * `w` - weeks
 * `y` - years
```

Comment:
This should also mention that years are always 365 days

Reply:
done, same for  days and weeks. That is what the comments on time.Duration also specify:
https://github.com/prometheus/common/blob/217fd62d44a79db8f842113234ae9383b8374983/model/time.go#L186-L187